Cubical/Categories/Displayed/Preorder.agda

```diff
@@ -0,0 +1,56 @@
+{-# OPTIONS --safe #-}
+module Cubical.Categories.Displayed.Preorder where
```

Comment:
Since this is analogous to `FullSubcategory` we should put in under `Constructions` too. Also for naming, if we like `FullSubcategory` then this could just be `Subcategory`

Reply:
In Section 9.8 of the HoTT book this is called a "notion of structure" over C, likely following https://ncatlab.org/nlab/show/stuff,+structure,+property terminology since the resulting projection is faithful.

Reply:
Yeah, that's a great idea! Structure over C instead of `Preorder` and property over C for `FullSubcategory`?

Reply:
By this logic, we should change Category^D to be `Stuff`...

Reply:
I can see the argument for renaming `FullSubcategory > PropertyOverC` and `Preoder > StructureOverC`, but this feels maybe out of scope for this PR. Shouldn't those changes also come with analogous changes to the non-displayed variants?